bindings/python/pyproject.toml

```diff
@@ -43,7 +43,7 @@ dependencies = [
     "cffi>=1.12.0,<2",
     "cryptography>=40",
     "packaging>=21.0",
-    "requests<3.0.0",
+    "httpx<0.25.0",
```

Comment:
The current version is 0.27, does that one not work for us?

Reply:
0.25.0 dropped support for Python 3.7, we need this until we upgrade to supporting >= 3.8.

Reply:
We can let pip handle that for us: https://github.com/encode/httpx/blob/4b85e6c3898b94e686b427afd83138c87520b479/pyproject.toml#L9